contracts/tribute/src/contract.rs

```diff
@@ -335,6 +326,62 @@ fn refund_tribute(
         }))
 }
 
+struct TopNProposalInfo {
+    pub top_n_proposal: Option<Proposal>,
+    pub is_above_voting_threshold: bool,
+}
```

Comment:
Would recommend adding quick docstrings to the struct and the fields,
explaining what to expect (in particular saying that it will be None, false if the prop is not in the top N)